.docs_overrides/main.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
```

Comment:
I see that the banner disappears from the page until a link is clicked again in two cases: 1) use the search function, or 2) click on a table of contents entry in the right column. Oh wait, it's just because the page scrolls down, but the banner shows up again if scrolling back to the top. I guess that behavior makes sense, it just confused me for a minute.

Reply:
Yes, the banner disappears on scroll down and appears again on scroll up.

Note: I have made the navigation tabs sticky, so that they might be more visible to new users.